pagopa/io-app
# feat: [#174866905] Detect CC brand and display CVV/CIN length accordingly

## Short description
creditCard utils now offers a new validation function able to return, along other validation info, the required CVV/CIN length (3 or 4 digits) of the detected brand.
AddCardScreen is modified accordingly.

## List of changes proposed in this pull request
- New validate function in creditCard
- Removal of old validation functions and related tests
- Update to AddCardScreen to use the object returned by validation
- Update to static label and placeholder string

## How to test
validation function is provided with its own set of unit tests, visual behavior is tested by hand on an IOS emulator (see attached image).
![ezgif com-video-to-gif-3](https://user-images.githubusercontent.com/8901084/98690401-2f7c1300-236d-11eb-85dd-8ced9cade4ac.gif)
